src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { PeopleFilters } from './PeopleFilters';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { Person } from '../../types/Person';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople)
+      .catch(() => setErrorMessage('Something went wrong'))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="columns is-desktop is-flex-direction-row-reverse">
+          <div className="column is-7-tablet is-narrow-desktop">
+            {people.length > 0 && (
+              <PeopleFilters />
+            )}
+          </div>
+
+          <div className="column">
+            <div className="box table-container">
+              {isLoading && (
+                <Loader />
+              )}
+
+              {errorMessage && (
+                <p data-cy="peopleLoadingError">
+                  {errorMessage}
+                </p>
+              )}
```

Comment:
```suggestion
              {!!errorMessage && (
                <p data-cy="peopleLoadingError">
                  {errorMessage}
                </p>
              )}
```
in rendering with &&  need to convert the value to boolean.